internal/handlers/pg/msg_find.go

```diff
@@ -119,6 +119,10 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 		}
 	}
 
+	if filter != nil && filter.Has("_id") {
+		sp.SqlFilters = must.NotFail(types.NewDocument("_id", must.NotFail(filter.Get("_id"))))
```

Comment:
Why we don't move that copy&pasted logic inside `QueryDocuments`? That's what we do in the Tigris handler

Reply:
Moved and simplified the code a bit for now.